plugins/nlp.h

```diff
@@ -30,24 +29,31 @@ struct ApplyStopWordsFunctionConfig {
 
 DECLARE_STRUCTURE_DESCRIPTION(ApplyStopWordsFunctionConfig);
 
+struct Words {
```

Comment:
They are like that so that the names match up with what they were previously called.  In other words, this is a refactoring of the same explicit structure that was present in the previous version, just put all in one place.   I agree we can improve things a lot, but for the moment we want to keep compatibility with previous usage of the functions and so we should keep them the same as what they were.  At least this way we have a place to document the parameters.